tools/py/Instanton_postproc.py

```diff
@@ -127,6 +128,11 @@
     raise ValueError(
         "We can not indentify the case. The valid cases are: 'reactant', 'TS' and 'instanton'"
     )
+if asr not in list(["poly", "crystal", "none"]):
```

Comment:
Do we issue any warnings if a molecule is linear and one is trying to use the "poly" mode? That is a rather general question but could be somewhat addressed also here and would impact the calculation of the instanton rates. As far as I remember, our "poly" mode assumes the molecule is _not_ linear.

Reply:
I added linear as a valid option for the asr variable.

Reply:
👍